kong.yml

```diff
@@ -100,7 +100,7 @@
 
   ######
   ## Name of the keyspace used by Kong. Will be created if it does not exist.
-  # keyspace: kong
+  keyspace: kong_dev
```

Comment:
I pushed this without it being ready to trigger the Travis tests, but I forgot to add the "WIP" label on the PR. It's now ready, and the issue we have is now that this is a patch introducing a breaking change because of the required change in kong.yml.